tests/ts_simple_show_create_table.erl

```diff
@@ -53,13 +59,25 @@ confirm() ->
     "dvv_enabled = false,\n"
     "dw = one,\n"
     "last_write_wins = true,\n"
-    "n_val = 2,\n"
+    "n_val = " ++ integer_to_list(NVal) ++ ",\n"
     "notfound_ok = true,\n"
     "postcommit = '',\n"
     "pr = 0,\n"
     "pw = 0,\n"
     "r = one,\n"
     "rw = one,\n"
-    "w = quorum)",
-    ?assertEqual(Expected, Got),
-    pass.
+    "w = quorum)".
+
+initialize(Table) ->
+    DDL = "CREATE TABLE " ++ Table ++ " ("
+    " somechars   VARCHAR   NOT NULL,"
+    " somebool    BOOLEAN   NOT NULL,"
+    " sometime    TIMESTAMP NOT NULL,"
+    " somefloat   DOUBLE,"
+    " PRIMARY KEY ((somechars, somebool, QUANTUM(sometime, 1, 'h')), "
+    " somechars, somebool, sometime)) "
+    " WITH (n_val=2)",
```

Comment:
Should this not also be a parameter, instead of hard-coded, if you changed it to one in `expected_show/2`? I guess it does not change on initialization...